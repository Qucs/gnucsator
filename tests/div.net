
Vac:V1 _net0 gnd U="1 V" f="" Phase="0" Theta="0"
R:R1 _net1 _net0 R="50" Temp="26.85" Tc1="0.0" Tc2="0.0" Tnom="26.85"
R:R2 _net1 gnd R="50" Temp="26.85" Tc1="0.0" Tc2="0.0" Tnom="26.85"
.TR:TR1 Type="lin" Start="0" Stop="1" Points="10" IntegrationMethod="Trapezoidal" Order="2" InitialStep="1 ns" MinStep="1u" MaxIter="150" reltol="0.001" abstol="1 pA" vntol="1 uV" Temp="26.85" LTEreltol="1e-3" LTEabstol="1e-6" LTEfactor="1" Solver="CroutLU" relaxTSR="no" initialDC="yes" MaxStep="0"
