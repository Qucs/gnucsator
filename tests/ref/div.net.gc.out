<Qucs Dataset>
default plugins: ignored
;: already installed, replacing
stashing as ;:0
*: already installed, replacing
stashing as *:0
': already installed, replacing
stashing as ':0
": already installed, replacing
stashing as ":0
spice_bjt: already installed, replacing
stashing as spice_bjt:0
spice_mos: already installed, replacing
stashing as spice_mos:0
Vac:V1 _net0 gnd U="1 V" f="" Phase="0" Theta="0" 
                         ^ ? V1: f has no value?
Gnucap   System status
iterations: op=0, dc=0, tran=35, fourier=0, total=47
transient timesteps: accepted=12, rejected=0, total=12
transient timesteps: accepted=0, rejected=0, total=0
nodes: user=2, subckt=0, model=0, total=2
dctran density=100.0%, ac density=100.0%

<indep time 12>
 0.
 0.01
 0.02
 0.03633333
 0.1239394
 0.2115455
 0.3429545
 0.4743636
 0.6057727
 0.7371818
 0.8685909
 1.
</indep>
<dep _net0.Vt time>
 0.
 0.06279051
 0.1253332
 0.2263113
 0.7023789
 0.9709522
 0.8342359
 0.1603824
-0.6167361
-0.9967584
-0.7350006
 0.
</dep>
<dep _net1.Vt time>
 0.
 0.03139526
 0.06266661
 0.1131556
 0.3511894
 0.4854761
 0.4171179
 0.08019118
-0.3083681
-0.4983792
-0.3675003
 0.
</dep>
